From 1b20dc0e651335c0ccb07e677e83c5d3fdbbc1c4 Mon Sep 17 00:00:00 2001
From: Alexander Courtis <alex@courtis.org>
Date: Sun, 5 Dec 2021 11:27:39 +1100
Subject: [PATCH] AMC do not resolve symlinks when finding

---
 autoload/nerdtree/ui_glue.vim | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/autoload/nerdtree/ui_glue.vim b/autoload/nerdtree/ui_glue.vim
index fc22f21..4ce308e 100644
--- a/autoload/nerdtree/ui_glue.vim
+++ b/autoload/nerdtree/ui_glue.vim
@@ -308,7 +308,8 @@ function! s:findAndRevealPath(pathStr) abort
     endif
 
     try
-        let l:pathStr = g:NERDTreePath.Resolve(l:pathStr)
+        " AMC please do not resolve symlinks
+        let l:pathStr = simplify(l:pathStr)
         let l:pathObj = g:NERDTreePath.New(l:pathStr)
     catch /^NERDTree.InvalidArgumentsError/
         call nerdtree#echoWarning('invalid path')
-- 
2.34.1

