From f6742c99a68f5cec3feceb204bb9c2145360144c Mon Sep 17 00:00:00 2001
From: Alexander Courtis <alex@courtis.org>
Date: Sun, 27 Feb 2022 11:19:15 +1100
Subject: [PATCH] amc black background

---
 sway/desktop/render.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/sway/desktop/render.c b/sway/desktop/render.c
index c088c936..e1ca1345 100644
--- a/sway/desktop/render.c
+++ b/sway/desktop/render.c
@@ -1093,7 +1093,8 @@ void output_render(struct sway_output *output, struct timespec *when,
 		render_unmanaged(output, damage, &root->xwayland_unmanaged);
 #endif
 	} else {
-		float clear_color[] = {0.25f, 0.25f, 0.25f, 1.0f};
+		// base00 for base16-bright
+		float clear_color[] = {0.0f, 0.0f, 0.0f, 1.0f};
 
 		int nrects;
 		pixman_box32_t *rects = pixman_region32_rectangles(damage, &nrects);
-- 
2.35.1

