From 9057668da7e6392436de15985422c6507f5d9f20 Mon Sep 17 00:00:00 2001
From: Alexander Courtis <alex@courtis.org>
Date: Tue, 21 Sep 2021 19:34:51 +1000
Subject: [PATCH] set nobuflisted on preview window

---
 autoload/gitgutter/hunk.vim | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/autoload/gitgutter/hunk.vim b/autoload/gitgutter/hunk.vim
index 28059c0..7b75ce7 100644
--- a/autoload/gitgutter/hunk.vim
+++ b/autoload/gitgutter/hunk.vim
@@ -489,7 +489,8 @@ function! s:open_hunk_preview_window()
   else
     let s:preview_bufnr = bufnr('')
   endif
-  setlocal filetype=diff buftype=acwrite bufhidden=delete
+  " AMC added nobuflisted
+  setlocal filetype=diff buftype=acwrite bufhidden=delete nobuflisted
   " Reset some defaults in case someone else has changed them.
   setlocal noreadonly modifiable noswapfile
   if g:gitgutter_close_preview_on_escape
-- 
2.33.0

